justfile

```diff
@@ -4,30 +4,21 @@ build:
 test:
     cargo test
 
+move-accessories-for-optimizer:
+    for folder in accessories/*; \
+    do \
+      folder=${folder/'accessories/'}; \
+      cp -R accessories/$folder contracts/$folder \
+    ;done
+
+remove-accessories-for-optimizer:
+    for folder in accessories/* \
+    do \
+    rm -R ../contracts/$folder \
+    done
+
```

Comment:
```suggestion
```

assuming this is leftover from an earlier iteration. this is done in the optimize.sh script?